src/js/main.js

```diff
@@ -363,6 +363,29 @@ if (document.body.dataset.page === "integrations") {
   }
 }
 
+// Add underline class to menu links on Mobile
+
+function setActiveMenuItem() {
+  const isMobile = window.matchMedia("(max-width: 1024px)").matches;
+
+  if (!isMobile) {
+    return;
+  }
+
+  const currentUrl = window.location.pathname;
+  const menuItems = document.querySelectorAll("nav a");
+
+  menuItems.forEach((item) => {
+    if (item.getAttribute("href") === currentUrl) {
+      item.classList.add("underline");
+    } else {
+      item.classList.remove("underline");
+    }
+  });
+}
+
+document.addEventListener("DOMContentLoaded", setActiveMenuItem);
```

Comment:
I think we don't need `DOMContentLoaded` because we load `main.js` at the bottom of the HTML page, so by that point all the content should be in place.

https://github.com/livebook-dev/livebook.dev/blob/7c3580392e4d638cdf0a7a2040e741bd0d9cc20e/src/partials/layout.html#L231

Conditional `classList.add`/`classList.remove` can usually be simplified into `classList.toggle` :)

Something like this:

```suggestion
const isMobile = window.matchMedia("(max-width: 1024px)").matches;

if (isMobile) {
  const currentUrl = window.location.pathname;
  const menuItems = document.querySelectorAll("nav a");
  
  menuItems.forEach((item) => {
    item.classList.toggle("underline", item.href === currentUrl);
  });
}
```

Reply:
Thanks =) I did some modifications and added some things José asked.